### PR TITLE
fix email typo Ref https://github.com/Giveth/giveth-dapp/issues/828

### PR DESCRIPTION
**Pull request by horus2021**
fixes https://github.com/Giveth/giveth-dapp/issues/828

### CODE REVIEW

File: src/utils/dappMailer.js [hunk 1]
```diff
@@ -243,9 +243,9 @@ module.exports = {
       title: 'Milestone review requested',
       image: 'Giveth-review-banner-email.png',
       text: `
-        <p><span style="line-height: 33px; font-size: 22px;">Hi ${data.user}</span></p>
+        <p><span style="line-height: 33px; font-size: 22px;">Hi ${data.user.toUpperCase()}</span></p>
```
Comment (aminlatifi) [inline src/utils/dappMailer.js:246]: You may should upper case just the first letter

File: src/utils/dappMailer.js [hunk 2]
```diff
@@ -243,9 +243,9 @@ module.exports = {
...
         <p>
-          The milestone <em>${data.milestoneTitle}</em> to the campaign <em>${data.campaignTitle}</em> has been marked as completed by the milestone owner.
+          The milestone <em>${data.milestoneTitle.toUpperCase()}</em> to the campaign <em>${data.campaignTitle.toUpperCase()}</em> has been marked as completed by the milestone owner.
```
Comment (aminlatifi) [inline src/utils/dappMailer.js:248]: Capitalize the first letters

**Comment by horus2021** [comment]
Hey @aminlatifi Check my updates on issue #828 again.

File: src/utils/dappMailer.js [hunk 3]
```diff
@@ -243,9 +243,9 @@ module.exports = {
       title: 'Milestone review requested',
       image: 'Giveth-review-banner-email.png',
       text: `
-        <p><span style="line-height: 33px; font-size: 22px;">Hi ${data.user}</span></p>
+        <p><span style="line-height: 33px; font-size: 22px;">Hi ${data.user(0).toUpperCase()+data.user.slice(1)}</span></p>
```
Comment (aminlatifi) [inline src/utils/dappMailer.js:246]: user is of type string, so user(0) doesn't have any meaning and creates exception!

File: src/utils/dappMailer.js [hunk 4]
```diff
@@ -243,9 +243,9 @@ module.exports = {
...
         <p>
-          The milestone <em>${data.milestoneTitle}</em> to the campaign <em>${data.campaignTitle}</em> has been marked as completed by the milestone owner.
+          The milestone <em>${data.milestoneTitle(0).toUpperCase()+data.milestoneTitle.slice(1)}</em> to the campaign <em>${data.campaignTitle(0).toUpperCase()+data.campaignTitle.slice(1)}</em> has been marked as completed by the milestone owner.
```
Comment (aminlatifi) [inline src/utils/dappMailer.js:248]: Here too, you can use data.milestoneTitle[0].toUpperCase() .... 
Also the code has eslint problem, you should add spaces before plus sign
Install some eslint plugin on you IDE

**Comment by aminlatifi** [comment]
The updates are not aligned with https://github.com/Giveth/giveth-dapp/issues/828 requirements!

**Comment by aminlatifi** [comment]
@horus2021 In the same file there is other instances of `milestone`, `campaign` and `owner` words which will be part of mail body. Please fix those too.

**Comment by aminlatifi** [comment]
LGTM!
Grate work @horus2021

**Pull request closed without merging**